requirements.txt

```diff
@@ -0,0 +1,3 @@
+coverage==3.7.1
+flake8==2.3.0
+flake8-import-order==0.5.1
```

Comment:
Maybe to [`setup_requires`](http://pythonhosted.org/setuptools/setuptools.html#new-and-changed-setup-keywords) in `setup.py`?